docs/vision-usage.rst

```diff
@@ -0,0 +1,190 @@
+Using the Vision API
+====================
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If you are Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.vision.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client()
+
+  or pass in ``credentials`` and ``project`` explicitly
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client(project='my-project', credentials=creds)
+
+Annotating an Image
+-------------------
+
+Annotate a single image
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
```

Comment:
Can probably omit setting the project explicitly, since it was recommended above to set the `GCLOUD_PROJECT` environment variable. Also makes it easier to copy/paste the sample code, since you won't have to hand-edit to delete "my-project-name" and add your own